style_editor.html

```diff
@@ -6302,7 +6302,7 @@
   constructor(MILLIS, POS) {
     super("WipeIn transition", arguments);
     this.add_arg("MILLIS", "INT", "Center Wipe time in milliseconds.");
-    this.add_arg("POS", "INT", "Position", 16384);
+    this.add_arg("POS", "TIME_FUNCTION", "Position", Int(16384)); // Updated POS to "TIME_FUNCTION"
```

Comment:
The comment doesn't really make any sense, please remove it.
(Comments like these belong in the PR description, not in the code.)